lib/utils/async.js

```diff
@@ -266,6 +268,7 @@ export const nextFrame = (() => {
 /* eslint-disable no-redeclare */
 // rest params can't be placed before non-rest params, so this is unfortunately necessary
 // add another case if you need more params
+declare function frameThrottle<A, B, C>(callback: (a: A, b: B, c: C, start: number) => void): () => void;
```

Comment:
Flow's intersections are order-dependent (so that it's predictable which case is chosen), so in general it's fine to put a more-specific case first. In this case I'd expect it to work however; maybe it's because the types are missing from the returned function? i.e. it's `() => void` instead of `(a: A, b: B, c: C) => void`.

Don't worry about it though, I'll be merging this next (probably) so I'll take care of it if that's the case.